animal-sniffer/build.gradle.kts

```diff
@@ -0,0 +1,54 @@
+import ru.vyarus.gradle.plugin.animalsniffer.info.SignatureInfoTask
+import ru.vyarus.gradle.plugin.animalsniffer.signature.BuildSignatureTask
+
+plugins {
+  id("otel.java-conventions")
+  id("ru.vyarus.animalsniffer")
+}
+
+val signatureJar = configurations.create("signatureJar") {
+  isCanBeConsumed = false
+  isCanBeResolved = false
+}
+val signatureJarClasspath = configurations.create("signatureJarClasspath") {
+  isCanBeConsumed = false
+  isCanBeResolved = true
+  extendsFrom(signatureJar)
+}
+val generatedSignature = configurations.create("generatedSignature") {
+  isCanBeConsumed = true
+  isCanBeResolved = false
+}
+configurations.add(signatureJar)
+configurations.add(signatureJarClasspath)
+configurations.add(generatedSignature)
+
+repositories {
+  mavenCentral()
+  google()
+}
+
+dependencies {
+  signature("com.toasttab.android:gummy-bears-api-21:0.6.1@signature")
+  signatureJar("com.android.tools:desugar_jdk_libs:2.0.4")
```

Comment:
I tested that it works by adding some code that uses `java.util.Base64` and then running the task `animalsnifferMain` which completed without any issues (since Base64 was added in version `2.0.4`). Then I changed this version to `2.0.3` and re-run the animal sniffer task which then failed.